benchmarks/dbscan/dbscan_timpl.hpp

```diff
@@ -273,6 +281,28 @@ bool ArborXBenchmark::run(ArborXBenchmark::Parameters const &params)
              ArborX_Benchmark::get_time("ArborX::HDBSCAN::dendrogram"));
       printf("---- edge sort      : %10.3f\n",
              ArborX_Benchmark::get_time("ArborX::Dendrogram::sort_edges"));
+      if (params.dendrogram == "alpha")
+      {
+        printf(
+            "---- alpha edges    : %10.3f\n",
+            ArborX_Benchmark::get_time("ArborX::Dendrogram::find_alpha_edges"));
+        printf("---- alpha vertices : %10.3f\n",
+               ArborX_Benchmark::get_time(
+                   "ArborX::Dendrogram::assign_alpha_vertices"));
+        printf("---- alpha matrix   : %10.3f\n",
+               ArborX_Benchmark::get_time(
+                   "ArborX::Dendrogram::build_alpha_incidence_matrix"));
+        printf("---- sided parents  : %10.3f\n",
+               ArborX_Benchmark::get_time(
+                   "ArborX::Dendrogram::update_sided_parents"));
+        printf("---- compression    : %10.3f\n",
```

Comment:
These timers are actually off, because our current extraction algorithms only grabs the first instance, so it would not accumulate them from multiple levels. Not 100% sure what to do here.